tutor/src/components/cc-dashboard/help.cjsx

```diff
@@ -28,28 +28,48 @@ CCDashboardHelp = React.createClass
         <p className="fine-print">This guide is always accessible from your main menu.</p>
         <div className="side-by-side">
           <div className="help">
-            <h3 className="improvements">
-              In January 2017, you’ll see the following improvements to Concept Coach:
-            </h3>
-
+            <b>
+              Add <CourseGroupingLabel plural {...glprops} /> to your course.
```

Comment:
Hm, good point, it can't hurt to do it. I actually just restored the old version from git here and didn't really read it very carefully.  It's weird that we'd used grouping label some places but not much.